src/app/common/components/AddTooltip.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { Tooltip } from '@patternfly/react-core';
+import { TooltipProps } from '@patternfly/react-core/dist/js/components/Tooltip/Tooltip';
```

Comment:
You should be able to just add TooltipProps to the import from `react-core` in the line above.

Reply:
I know but it doesn't work for me: 

`Module '"../../../../node_modules/@patternfly/react-core/dist/js"' has no exported member 'TooltipProps'.ts(2305)`

Reply:
Aha! Looks like that was a bug in PF, it was supposed to be exported and they fixed it in a newer release. if I update PF on my end it fixes it. Sounds like that's out of scope for this PR though so we can leave this